openbas-front/src/admin/components/common/injects/InjectPopover.tsx

```diff
@@ -48,6 +68,8 @@ const InjectPopover: FunctionComponent<Props> = ({
   const [_injectTestResult, setInjectTestResult] = useState<InjectStatus | null>(null);
   const [anchorEl, setAnchorEl] = useState<Element | null>(null);
 
+  const isExercise = useHelper((helper: ExercisesHelper) => helper.getExercisesMap()[exerciseOrScenarioId!] === undefined);
+
```

Comment:
Here you could make _different from undefined_ in order to know if the id is an exerciseId, or you could call this const: is NotExercise